packages/core/nest-factory.ts

```diff
@@ -157,8 +163,14 @@ export class NestFactoryStatic {
     container: NestContainer,
     config = new ApplicationConfig(),
     httpServer: HttpServer = null,
+    appOptions?: NestApplicationOptions,
   ) {
     const instanceLoader = new InstanceLoader(container);
+
+    if (appOptions && appOptions.disableInstanceLoaderLogs) {
+      instanceLoader.disableLogs();
+    }
+
```

Comment:
Instead of marking this as optional, could we give it a default value?

```suggestion
    appOptions: NestApplicationOptions = {
      disableInstanceLoaderLogs: false
    },
  ) {
    const instanceLoader = new InstanceLoader(container);

    if (appOptions.disableInstanceLoaderLogs) {
      instanceLoader.disableLogs();
    }
```

Also, do we need the full options here, or just the `disableInstanceLoaderLogs` option? We might be able to cut this down to a simple `boolean` instead of the full object